From 948a9618bb75ac3dede70567b8d2337dacb7d881 Mon Sep 17 00:00:00 2001
From: Andoni Morales Alastruey <ylatuya@gmail.com>
Date: Mon, 8 Apr 2013 15:28:41 +0200
Subject: [PATCH 14/14] libdv: fix build of gasmoff after autoreconf

This makes it build with -DHAVE_CONFIG_H and sets the correct
endianess building after autoreconf
---
 libdv/Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/libdv/Makefile.am b/libdv/Makefile.am
index 05f23da..9b9132c 100644
--- a/libdv/Makefile.am
+++ b/libdv/Makefile.am
@@ -15,8 +15,7 @@ if HOST_X86_64
 GASMOFF=gasmoff
 endif # HOST_X86_64
 
-
-noinst_PROGRAMS= #dovlc testvlc testbitstream $(GASMOFF) recode reppm enctest
+noinst_PROGRAMS= $(GASMOFF) #dovlc testvlc testbitstream $(GASMOFF) recode reppm enctest
 
 #
 # If HOST_X86 is set, we build all the x86 asm stuff..
-- 
2.1.4

